app/models/appci.py

```diff
@@ -46,6 +46,13 @@ def init():
                           display_name='Unstable (x86)',
                           url='https://ci-apps-unstable.yunohost.org/ci/logs/list_level_unstable.json',
                           url_per_app='https://ci-apps-unstable.yunohost.org/ci/apps/{}/')
+        
+        yield AppCIBranch(name='buster',
+                          arch="x86",
+                          branch="buster",
+                          display_name='buster (x86)',
+                          url='https://ci-stretch.nohost.me/ci/logs/list_level_stable.json',
+                          url_per_app='https://ci-stretch.nohost.me/ci/apps/{}/')
```

Comment:
As a somewhat unrelated note, could we please change/use another domain name, like, anything except the exact opposite of what the server is (iirc, this server was previously about the jessie CI, and now it's for buster, yet it's called stretch...).

Reply:
That could be indeed a great things :D